pkg/resourceconverter/resource_converter_test.go

```diff
@@ -34,19 +34,21 @@ func TestReflectionConverter(t *testing.T) {
 			WeirdTags: []WeirdTags{{WeirdKey: "key1", WeirdValue: "val1"}, {WeirdKey: "key2", WeirdValue: "val2"}},
 		}
 		rC := &ReflectionConverter{
-			IdField:      "ID",
-			TagField:     TagField{Name: "WeirdTags", Key: "WeirdKey", Value: "WeirdValue"},
-			ResourceType: "DummyResource",
-			Region:       "dummyRegion",
+			IdField:        "ID",
+			DisplayIdField: "Attr2",
```

Comment:
awesome, while you are at it, can you please add the test case where DisplayIdField is missing?